src/timeline.js

```diff
@@ -357,7 +372,22 @@ export default class Timeline extends React.Component {
      * ```
      * @type { JSX.Element }
      */
-    backgroundLayer: PropTypes.object
+    backgroundLayer: PropTypes.object,
+
+    /**
+     * Function called when dragToCreateMode == true on dragend
+     * @param {number} groupIndex Index of selected group
+     * @param {number | string} itemIndex Index for new item
+     * @param {number | object} itemStart Start for new item
+     * @param {number | object} itemEnd End for new item
+     * @type { Function }
+     */
+    onDragToCreateEnded: PropTypes.func,
+
+    /**
+     * @type { boolean }
+     */
+    renderMenuButton: PropTypes.bool
```

Comment:
nu-mi dau seama de ce ar fi nevoie de ac prop. In caz de "urgenta", omul poate suprascrie functia de renderbutton (care btw, se numeste la fel cu ac prop; poate fi misleading) si sa zica "return null"